### PR TITLE
[EP] Ignore service emails from microsoft

### PR DESCRIPTION
**Pull request by sajjadmurtaza**

### CODE REVIEW

File: Backend/test/email/handle_mail_test.py
[the remark this answers is not in the document]
Comment (irhox) [reply Backend/test/email/handle_mail_test.py:1]: @M-HRL what do you mean? I wrote a unit test for the method that was creatd

[the remark this answers is not in the document]
Comment (M-HRL) [reply Backend/test/email/handle_mail_test.py:1]: @irhox, yeah, but only wether the email can be process cause it is not blacklisted. You could simply reach a better test coverage if you write a second test method that ensures that blacklisted emails are filtered out and therefore not processed.

[the remark this answers is not in the document]
Comment (irhox) [reply Backend/test/email/handle_mail_test.py:1]: @M-HRL i added a new unit test, look into if its what you meant

**Comment by M-HRL** [comment]
Looks fine. Good work, @irhox!
Bildschirmfoto 2024-01-23 um 10 11 11

**Review by M-HRL — APPROVED** [review]
@irhox, you can merge it yourself.

**Pull request merged**